inc/class-wpseo-utils.php

```diff
@@ -1135,13 +1135,30 @@ public static function is_plugin_network_active() {
 		return $network_active;
 	}
 
+	/**
+	 * Gets the type of the current page.
+	 *
+	 * @return string Returns 'post' if the current page is a post edit page. Taxonomy in other cases.
+	 */
+	public static function get_page_type() {
+		global $pagenow;
+		if ( WPSEO_Metabox::is_post_edit( $pagenow ) ) {
+			return 'post';
+		}
+
+		return 'taxonomy';
+	}
+
 	/**
 	 * Getter for the Adminl10n array. Applies the wpseo_admin_l10n filter.
 	 *
 	 * @return array The Adminl10n array.
 	 */
 	public static function get_admin_l10n() {
-		$wpseo_admin_l10n = [];
+		$wpseo_admin_l10n = [
+			'isPremium' => WPSEO_Utils::is_yoast_seo_premium(),
+			'isPostType' => WPSEO_Utils::get_page_type() === 'post',
```

Comment:
@omarreiss does this comment refer to `isPostType` or also to `isPremium`?

Because if I move `isPremium` to the Post/Term L10n objects, I have to do this in the metabox:

```js
const isPremium = get( window, [ "wpseoPostScraperL10n.isPremium" ], false ) || get( window, [ "wpseoTermScraperL10n.isPremium" ], false );
```

For `isPostType` it works fine to just check whether `window.wpseoPostScraperL10n` is defined.

Reply:
Only isPostType. I would've commented the other line as well otherwise.

Reply:
Fair enough 👍